<launch>
    <arg name="config_pkg" default="$(eval find('kingfisher_description'))"/>
    <arg name="debug" default="none" />
    <arg name="model_name" default="kingfisher"/>
    <arg name="namespace" default="kf"/>

    <param name="tf_namespace" value="$(arg namespace)" />
     <node pkg="tf2_ros" type="static_transform_publisher" name="vehicle_t_sonar" args="0.45 0 0 0 0 0 $(arg namespace)/vehicle $(arg namespace)/sonar1"/>
    <node pkg="tauv_common" type="sonar_map" name="sonar_map" output="screen">
        <rosparam command="load" file="$(arg config_pkg)/yaml/sonar.yaml" />
    </node>
</launch>